blocks/additional-info/__subtitle/additional-info__subtitle.css

```diff
@@ -15,5 +15,6 @@
   .additional-info__subtitle {
     font-size: 18px;
     line-height: 18px;
+    margin: 22px 0 0;
```

Comment:
In 320px screen the size of the subtitle is 12px

Reply:
switched to content_subtitle